automation/build-bin.ts

```diff
@@ -382,6 +383,7 @@ export async function buildOclifInstaller() {
 		if (process.platform === 'win32') {
 			await signWindowsInstaller();
 		}
+		await afterSignHook.sign(path.join(ROOT, 'dist', renamedOclifInstallers.darwin)); // File to notarize
```

Comment:
I think the path is incorrect here, because `renamedOclifInstallers.darwin` already includes ROOT/dist.